tools/corals/multilocus_genotype/multilocus_genotype.R

```diff
@@ -232,42 +248,193 @@ report_user <- pi %>%
         by="affy_id") %>%
     mutate(DB_match = ifelse(is.na(DB_match), "failed", DB_match))%>%
     mutate(coral_mlg_clonal_id = ifelse(is.na(coral_mlg_clonal_id), "failed", coral_mlg_clonal_id)) %>%
+    mutate(genetic_coral_species_call=ifelse(percent_alternative_coral >= 40 & percent_alternative_coral<= 44.5,"A.palmata","other")) %>%
+    mutate(genetic_coral_species_call=ifelse(percent_alternative_coral >= 45.5 & percent_alternative_coral<= 50,"A.cervicornis",genetic_coral_species_call)) %>%
+    mutate(genetic_coral_species_call=ifelse(percent_heterozygous_coral > 40,"A.prolifera",genetic_coral_species_call)) %>%
     ungroup() %>%
     select(-group);
 
 write.csv(report_user, file=opt$output_stag_db_report, quote=FALSE);
 
-# Combine sample information for database.
-report_db <- pinfo %>%
-    left_join(report_user %>%
-        select("user_specimen_id", "affy_id", "coral_mlg_clonal_id", "DB_match",
-               "percent_missing_data_coral", "percent_mixed_coral", "percent_reference_coral",
-               "percent_alternative_coral"),
-        by="user_specimen_id");
-
-# Create vector indicating number of individuals desired
-# made from affy_id collumn of report_user data table.
-i <- report_user[[2]];
-sub96 <- obj2[i, mlg.reset=FALSE, drop=FALSE];
+# Database tables
+## Sample.table
+sample_db <- pinfo %>%
+  left_join(
+    report_user %>%
+      select("user_specimen_id","affy_id",
+             "percent_missing_data_coral","percent_heterozygous_coral","percent_reference_coral",
+             "percent_alternative_coral"),
+    by='user_specimen_id');
+
+###representative clone for genotype.table
```

Comment:
Comments should start with a single # character, and a space should separate the # from the first word.  Comments should be complete, grammatically correct sentences.